docs/documentation/getting-started/load.mdx

```diff
@@ -50,3 +61,5 @@ pg_restore --verbose --clean --no-acl --no-owner \
     -Fc \
     old_db.dump
 ```
+
+Congratulations! You are now ready to run real queries over your data. To get started, refer to our [indexing documentation](https://docs.paradedb.com/documentation/indexing/create_index).
```

Comment:
This should redirect to the quickstart which is actually in the "getting started" section.

Reply:
Users come to this from the quickstart, though.

Install --> Quickstart --> Load your data --> Start writing your own queries

Reply:
Then they should be taken to the "full text search" page, not the "create an index" page (which they've already done).

Reply:
Done